### DISCUSSION TITLE
Catch-all mailbox configuration

### DISCUSSION THREAD

**Discussion by arisudesu** [emersion/maddy#364] [Q&A] [answered]
Hello,

Is it possible to configure server to accept mail to non-existent aliases, and deliver it to single alias?

What I'm trying to achieve is to implement catch-all mailbox, while also having some ordinary aliases. E.g.:
`postmaster@domainname` -> `postmaster@domainname`  # for communication with servers
`inbox@domainname` -> `inbox@domainname`  # for communication with humans
`*@domainname` -> `forspam@domainname`  # anything else goes here

I've seen it asked and answered in #243, though it is unclear to me how the final configuration should look and where the changes should be placed.

Thanks.

**Comment by foxcpp** [comment, marked as answer]
The configuration snippet provided in #243 is supposed to replace this part of the default configuration:
https://github.com/foxcpp/maddy/blob/2d691f1f7c2463db78e9ee97318d7e2a58f2a5f3/maddy.conf#L97-L99
making it
```
source $(local_domains) {
    reject 501 5.1.8 "Use Submission for outgoing SMTP"
}
default_source {
    destination_in &local_mailboxes {
        deliver_to &local_mailboxes
    }
    destination $(local_domains){
        modify {
            replace_rcpt regexp ".*" "forspam@domainname"
        }
        deliver_to &local_mailboxes
    }

    default_destination {
        reject 550 5.1.1 "User doesn't exist"
    }
}
```

**Reply by arisudesu** [reply]
Thank you, it works!
